in_text/text_data_release.yml

```
@@ -512,7 +518,7 @@ entities:
     -
       attr-label: sink
       attr-def: >-
-        XX
+        sink variable (Y) used for information theory calculations
```

is this the name of the variable or data?
```suggestion
        Sink variable (Y) used for information theory calculations
```